packages/core/src/VRange.ts

```diff
@@ -10,6 +17,51 @@ export class VRange {
     // Public
     //--------------------------------------------------------------------------
 
+    /**
+     * Describe a collapsed range at the given location, targetting a reference
+     * VNode and specifying the position relative to that VNode.
+     *
+     * @param reference
+     * @param position
+     */
+    static at(reference: VNode, position = RelativePosition.BEFORE): RangeDescription {
+        return VRange.select(reference, position, reference, position);
+    }
+    /**
+     * Describe a range selecting the given nodes.
+     * Consider `ab` (`[` = start, `]` = end):
+     * `select(a)` => `[a]b`
+     * `select(a, b)` => `[ab]`
+     * `select(a, RelativePosition.BEFORE, b, RelativePosition.BEFORE)` => `[a]b`
+     * `select(a, RelativePosition.AFTER, b, RelativePosition.BEFORE)` => `a[]b`
+     * `select(a, RelativePosition.BEFORE, b, RelativePosition.AFTER)` => `[ab]`
+     * `select(a, RelativePosition.AFTER, b, RelativePosition.AFTER)` => `a[b]`
+     *
+     * @param startNode
+     * @param [startPosition] default: `RelativePosition.BEFORE`
+     * @param [endNode] default: `startNode`
+     * @param [endPosition] default: `RelativePosition.AFTER`
+     */
+    static select(startNode: VNode, endNode?: VNode): RangeDescription;
```

Comment:
`getRangeDescription`?
I first thought that select would actually do something in the VDOM. Then I saw it is a static function and the only thing it does is return an object.
`select` implies a mutation somewhere.